src/shared/metrics.tsx

```diff
@@ -47,6 +48,10 @@ interface EnToggleMetric extends Metric {
   state: boolean;
 }
 
+interface ClearExposedMetric extends Metric {
+  timeamount: number;
```

Comment:
I think this should have the units in it, maybe `hoursSinceExposureDetectedAt` or something along those lines.